static/app/routes.tsx

```diff
@@ -1742,6 +1756,12 @@ function buildRoutes() {
     </Route>
   );
 
+  const customerDomainorganizationRoutes = (
```

Comment:
```suggestion
  const customerDomainOrganizationRoutes = (
```

Casing looked off here.